cli/dts/lib.deno.worker.d.ts

```diff
@@ -7,8 +7,9 @@
 /// <reference lib="deno.shared_globals" />
 /// <reference lib="esnext" />
 
-declare interface DedicatedWorkerGlobalScope {
-  self: DedicatedWorkerGlobalScope & typeof globalThis;
+declare class WorkerGlobalScope extends EventTarget {
+  new(): WorkerGlobalScope;
+  self: WorkerGlobalScope & typeof globalThis;
```

Comment:
I'm a bit puzzeled here... [MDN says](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope) that `WorkerGlobalScope` is an interface so it shouldn't be available as a value?

Also in `99_main.js` you add `WorkerGlobalScope` as `self.WorkerGlobalScope` but it's not reflected in these definition.

Reply:
I believe MDN wouldn't have a page for it if it didn't exist as an object. I've checked that it exists on Chrome and Firefox in workers. Lots of classes like this including `Window` are called interfaces in the spec but exist as objects so the prototypes are kept somewhere.

> Also in `99_main.js` you add `WorkerGlobalScope` as `self.WorkerGlobalScope` but it's not reflected in these definition.

It's `declare`d at the top-level.

Reply:
There we go: https://heycam.github.io/webidl/#es-interfaces:
> For every interface that is exposed in a given Realm and that is not declared with the [LegacyNoInterfaceObject] or [LegacyNamespace] extended attributes, a corresponding property exists on the Realm's global object. The name of the property is the identifier of the interface, and its value is an object called the interface object. The characteristics of an interface object are described in § 3.7.1 Interface object.